main.go

```diff
@@ -15,19 +15,24 @@ import (
 func Digest(ctx context.Context, e firestore.DocumentEventData) error {
 	log.Printf("→ firestore protobuf document: %v", e)
 
-	b, err := firestoreconv.Report(e.Value)
+	r, err := firestoreconv.Report(e.Value)
 	if err != nil {
 		return err
 	}
 
-	data := b.Benchmark.Times()
+	data := r.Benchmark.Times()
 
 	s, err := stats.Compute(data)
 	if err != nil {
 		return err
 	}
 
-	log.Printf("→ computed stats: %v", s)
+	b, err := s.MarshalJSON()
```

Comment:
[question] Why do we MarshalJSON the computed stats?

Reply:
Because by using `float64` as underlying data type we lose the pretty print like `123µs`.
I was experimenting with this to pretty-log it in cloud functions.

May not stay.

Reply:
I wouldn't log successful computes as it's a lot of pollution, however we can keep the pretty print for debugging in the errors above?

Reply:
I agree as the end goal, but for now as there is not sql data connected to worker, the output is lost if not logged somewhere. This way we can at least see the output in Cloud console.

I expect this to change with #5.
What do you think ?

Reply:
Yup makes sense 👍